source/tools/file_to_pascal_string_Tokyo.dproj

```diff
@@ -1,220 +0,0 @@
-﻿<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">

```

Comment:
Same as above: do not delete Delphi Tokyo files.

Reply:
Same reason here.